test/e2e/windows/gmsa_full.go

```diff
@@ -420,3 +493,16 @@ func runKubectlExecInNamespace(namespace string, args ...string) (string, error)
 	namespaceOption := fmt.Sprintf("--namespace=%s", namespace)
 	return framework.RunKubectl(namespace, append([]string{"exec", namespaceOption}, args...)...)
 }
+
+func getGmsaDomainIP(f *framework.Framework, podName string) string {
```

Comment:
Is there any reason we are using the IP address here instead of the dns name of the doman controller?  We had an issue in the past where there was a bug with the DNS name but ip worked fine: https://github.com/kubernetes-sigs/windows-gmsa/issues/30.

Reply:
Yeah, thanks for pointing this out. I'm using AD 2016 so dns name doesn't work for me. I tried the fix mentioned in the issue but it broke my testbed. I'll spend more time on this and follow up on the refactor pr.